src/MixedModelsMakie.jl

```diff
@@ -1,5 +1,13 @@
 module MixedModelsMakie
+    using CairoMakie
```

Comment:
Let's leave this as is for now, but a note for our future selves is to allow users to choose their own backend.

Reply:
It seems that Figure and Axis are defined in `AbstractPlotting.jl`.  Should we make the change now to have this package depend on `AbstractPlotting` and encourage the user to use their own back end package?

Reply:
Yes, that seems like a good idea.